lib/find-view.coffee

```diff
@@ -447,21 +447,27 @@ class FindView extends View
     else
       optionButton.removeClass 'selected'
 
+  anyMarkersAreSelected: =>
+    selections = @model.getEditor()?.getSelections() or []
+    _.any selections, (selection) =>
+      _.any @model.markers or [], (marker) =>
+        _.isEqual(marker.getBufferRange(), selection.getBufferRange())
```

Comment:
@colllin I think that we need to avoid a `selections.length x markers.length` loop here, mainly because there can be hundreds of thousands of markers. I think the performance will be more predictable if you query for the markers by range, using `TextEditor::findMarkers`. There is already code to do this in [`BufferSearch.prototype.findMarker`](https://github.com/atom/find-and-replace/blob/d719c79b358cf59e15ca185091965d29a9ead49b/lib/buffer-search.coffee#L194). You could use that method like this:

```coffee
for selectedRange in @model.getEditor().getSelectedBufferRanges()
  if @model.findMarker(selectedRange)
    return true
```